src/OpenSage.Game/Diagnostics/GameLoopView.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ImGuiNET;
+
+namespace OpenSage.Diagnostics
+{
+    internal class GameLoopView : DiagnosticView
+    {
+        public GameLoopView(DiagnosticViewContext context) : base(context)
+        {
+
+        }
+
+        public override string DisplayName => "Game loop";
+
+        protected override void DrawOverride(ref bool isGameViewFocused)
+        {
+            ImGui.Text($"Logic frame: {Game.CurrentFrame}");
+            ImGui.Separator();
+            ImGui.Text($"Map time:    {Game.MapTime.TotalGameTime}");
```

Comment:
Shall we round these values to, say, 2 decimal places? I find it a bit hard to read at a glance.